src/genie/libs/parser/iosxe/show_hw_module_status.py

```diff
@@ -0,0 +1,116 @@
+''' show_hw_module_status.py
+IOSXE parsers for the following show commands:
+    * show hw_module subslot {subslot} transceiver {transceiver} status
+'''
+
+import re
+
+from genie.metaparser import MetaParser
+# from genie.metaparser.util.schemaengine import Any, Optional
+
+# parser utils
+# from genie.libs.parser.utils.common import Common
+
+
+# ==========================
+# Schema for:
+#  * 'show_hw_module_status'
+# ==========================
+class Show_Hw_Module_StatusSchema(MetaParser):
+    """Schema for show_hw_module_status."""
+
+    schema = {
+    "transceiver_status": {
+        "slot_id": int,
+        "subslot": int,
+        "port_id": 0,
+        "module_temperature": str,
+        "supply_voltage_mVolts": str,
+        "bias_current_uAmps": int,
+        "tx_power_dBm": str,
+        "optical_power_dBm": str
+    }
+}
+
+
+# ==========================
+# Parser for:
+#  * 'show_hw_module_status'
```

Comment:
can you put the actual show command here

Reply:
added the actual command.